babel/src/test/java/org/apache/calcite/test/BabelParserTest.java

```diff
@@ -1877,6 +1877,26 @@ private void checkParseInfixCast(String sqlType) {
     sql(sql).ok(expected);
   }
 
+  @Test void testAlterAddColumn() {
+    final String sql = "alter table foo add bar integer";
+    final String expected = "ALTER TABLE `FOO` ADD (`BAR` INTEGER)";
+    sql(sql).ok(expected);
+  }
+
+  @Test void testAlterAddMultiColumns() {
+    final String sql = "alter table foo add (bar integer, baz integer)";
+    final String expected = "ALTER TABLE `FOO` ADD"
+        + " (`BAR` INTEGER, `BAZ` INTEGER)";
+    sql(sql).ok(expected);
+  }
+
+  @Test void testAlterAddColumnWithTableAttribute() {
+    final String sql = "alter table foo, no fallback add bar integer";
+    final String expected = "ALTER TABLE `FOO`, NO FALLBACK ADD"
+        + " (`BAR` INTEGER)";
+    sql(sql).ok(expected);
+  }
+
   @Test void testIndexWithoutName() {
```

Comment:
Let's add a failing test for 0 columns passed in.

Reply:
Maybe it would actually make sense to add 2 test cases (one for the case without parentheses, and one with)?